app/api/helpers/permissions.py

```diff
@@ -87,5 +87,127 @@ def decorated_function(*args, **kwargs):
 
     return decorated_function
 
+@second_order_decorator(jwt_required)
+def is_organizer(f):
+    """
+    Allows only Organizer to access the event resources.
+    :param f:
+    :return:
+    """
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
+
+        if user.is_staff:
+            return f(*args, **kwargs)
+        if 'event_id' in kwargs and user.is_organizer(kwargs['event_id']):
+            return f(*args, **kwargs)
+        return ForbiddenError({'source': ''}, 'Organizer access is required').respond()
+
+    return decorated_function
+
+@second_order_decorator(jwt_required)
+def is_coorganizer(f):
+    """
+    Allows Organizer and Co-organizer to access the event resources.
+    :param f:
+    :return:
+    """
+    @wraps(f)
+    def decorated_function(*args, **kwargs):
+        user = current_identity
+
+        if user.is_staff:
+            return f(*args, **kwargs)
+        if 'event_id' in kwargs and (
+                user.is_coorganizer(kwargs['event_id']) or 
+                user.is_organizer(kwargs['event_id']) ):
+                return f(*args, **kwargs)
```

Comment:
Thanks! Strange that PyCharm didn't throw this error. Fixes all PEP8 issues as well.